JSDOMParser.js

```diff
@@ -875,7 +875,11 @@
 
   JSDOMParser.prototype = {
     error: function(m) {
-      dump("JSDOMParser error: " + m + "\n");
+      if (typeof dump !== "undefined") {
```

Comment:
My kingdom for an eslint rule that complains about the use of triple equals, but I guess I just have to accept that I have lost that fight with the internet... I won't bother you with my old-man-get-off-my-lawn-yelling, I should probably blog about it at some point.

Reply:
I'm in the triples-everywhere camp because of past traumas :P But yeah, an eslint rule that points out unnecessary triples would be awesome.